benchmark/stream_buffer.cpp

```diff
@@ -63,6 +63,30 @@ StreamBuffer_Front(benchmark::State& state)
     }
 }
 
+static void
+StreamBuffer_Pop(benchmark::State& state)
+{
+    quicr::StreamBuffer<std::uint8_t> buffer;
+    std::vector<uint8_t> bytes(1'000'000'000, 0);
+    buffer.Push(bytes);
+
+    for ([[maybe_unused]] const auto& _ : state) {
+        buffer.Pop();
+    }
+}
+
+static void
+StreamBuffer_PopBytes(benchmark::State& state)
+{
+    quicr::StreamBuffer<std::uint8_t> buffer;
+    std::vector<uint8_t> bytes(1'000'000'000, 0);
```

Comment:
Same, is there a reason for this value?

Reply:
Because there needs to be a value to pop to test it, and the max iterations for tests (from observation only) is about that much.